quinn-proto/src/connection/mod.rs

```diff
@@ -2942,12 +3072,12 @@ where
             .saturating_sub(self.in_flight.bytes)
     }
 
-    /// Whether no timers but keepalive and idle are running
+    /// Whether no timers but keepalive, idle and pushnewcid are running
```

Comment:
Looks like we no longer have the idle timer, so let's remove it from this comment?

Reply:
Not sure I got what you mean, but from the latest code in mainline branch, the idle timer is still there.

https://github.com/quinn-rs/quinn/blob/main/quinn-proto/src/connection/timer.rs#L7-L8
and
https://github.com/quinn-rs/quinn/blob/main/quinn-proto/src/connection/mod.rs#L2956

Reply:
I was confused by the initial filter call only referencing `KeepAlive` and `PushNewCid`, but you're right. I think the comment here is not great, but that's a preexisting problem, I file a separate issue to clean it up after this PR lands.